src/services/edusharing/logic/connector.js

```diff
@@ -0,0 +1,165 @@
+const request = require('request-promise-native');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected 1 empty line after require statement not followed by another require.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4791331)